index.html

```diff
@@ -5,10 +5,21 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles.css">
+    @import url('https://fonts.googleapis.com/css2?family=Bungee+Inline&display=swap');
```

Comment:
Not sure if this is needed in both the html and css, maybe the link bar goes in here, however we can test that later,

Great work 🌟

Reply:
I agree, I think you only need it to go either in the head of the HTML or import through CSS.